src/assets/svg/4G.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="41" height="40" viewBox="0 0 41 40" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path d="M20.7841 30.234C22.4941 30.234 24.5101 29.82 26.2561 28.218L27.1381 22.854H22.3321L21.9361 25.23H23.7541L23.4841 26.904C22.8901 27.282 22.2601 27.426 21.5221 27.426C19.8481 27.426 19.1101 26.31 19.1101 24.546C19.1101 21.702 20.3881 19.686 22.5841 19.686C23.8981 19.686 24.2401 20.424 24.1861 21.342H27.3901C27.6241 18.768 26.5801 16.824 22.9981 16.824C18.1021 16.824 15.7981 20.604 15.7981 25.05C15.7981 27.858 17.0761 30.234 20.7841 30.234Z" fill="#484848"/>
```

Comment:
use stroke="currentColor"
Applies to all strokws in all three files

Reply:
Can't believe I missed that 😅

Reply:
I can see on some of the other files it also uses fill="currentColor". Is it only if fill has a color or also if fill is fill="none"? :)

Reply:
fill="none" needs to stay as is. Only places where "#484848" is present we should change it to currentColor 😉

Reply:
should be fixed now 😊